Makefile

```
@@ -1,7 +1,10 @@
 # OpenEdx E-Commerce
-EDX_EC_ARCHIVE_URL ?= https://github.com/edx/ecommerce/archive/master.tar.gz
-EDX_EC_RELEASE_REF ?= master
-EDX_EC_DOCKER_TAG  ?= latest
+EDX_EC_RELEASE               ?= master
```

Since it is the name of your image...
```suggestion
EDXEC_RELEASE               ?= master
```
And everywhere else?

---

Yup.